### DISCUSSION TITLE
Are screws and nuts necessary for a custom robot

### DISCUSSION THREAD

**Discussion by huiduan-25** [cyberbotics/webots#5219] [Q&A]
Hello,

I import a robot assembly from Solidworks into Webots to run some simulations. Since all the matches in SolidWorks are gone, should I also delete the screws and nuts and just keep the necessary components?

Best wishes

Xiangyu

**Comment by omichel** [comment]
Screw and nuts are actually ignored by Webots physics engine unless they have some specific bounding objects.
